igvm_defs: build error without "unstable" feature

`cargo build` of igvm_defs fails:
```
error[E0599]: no associated item named `SEV` found for struct `IgvmPlatformType` in the current scope
   --> src/lib.rs:383:5
    |
368 | #[open_enum]
    | ------------ associated item `SEV` not found for this struct
...
383 |     SEV = 0x04,
    |     ^^^ associated item not found in `IgvmPlatformType`

error[E0599]: no associated item named `SEV_ES` found for struct `IgvmPlatformType` in the current scope
   --> src/lib.rs:387:5
    |
368 | #[open_enum]
    | ------------ associated item `SEV_ES` not found for this struct
...
387 |     SEV_ES = 0x05,
    |     ^^^^^^ associated item not found in `IgvmPlatformType`

For more information about this error, try `rustc --explain E0599`.
```

If works, if the `unstable` feature is set: `cargo build -F unstable`

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,7 +16,7 @@ anyhow = "1.0"
 bitfield-struct = "0.7"
 crc32fast = { version = "1.3.2", default-features = false }
 hex = { version = "0.4", default-features = false }
-open-enum = "0.5.1"
+open-enum = "0.5.2"
 range_map_vec = "0.2.0"
 static_assertions = "1.1"
 thiserror = "1.0"

